po/fa.po

```diff
@@ -304,11 +304,11 @@ msgstr "متدها"
 
 #: src/SUMMARY.md src/methods-and-traits.md src/methods-and-traits/traits.md:1
 msgid "Traits"
-msgstr "صفت‌ها"
+msgstr ""
```

Comment:
Dont leave empty any mgstr

Reply:
> به این دلیل میگین msgstr رو خالی نزارم که شخص دیگه ای این رو ترجمه نکنه؟

That's because if it's empty, it probably shows the whole text untranslated.

Reply:
To appearance in translated language menu should complete all msgstr